pkgs/development/python-modules/geopy/default.nix

```diff
@@ -1,31 +1,64 @@
 { stdenv
 , buildPythonPackage
 , fetchPypi
+, fetchFromGitHub
 , isPy27
+, isPy37
 , mock
 , tox
 , pylint
+, geographiclib
 }:
 
-buildPythonPackage rec {
+let
   pname = "geopy";
-  version = "1.20.0";
-  disabled = !isPy27;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "9419bc90ee6231590c4ae7acf1cf126cefbd0736942da7a6a1436946e80830e2";
+  commonBuildInputs = [ mock tox pylint ];
+
+  geopy27 = rec {
+    version = "1.20.0";
+    disabled = !isPy27;
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "9419bc90ee6231590c4ae7acf1cf126cefbd0736942da7a6a1436946e80830e2";
+    };
+
+    buildInputs = commonBuildInputs;
+    propagatedBuildInputs = [];
   };
 
-  doCheck = false;  # too much
+  geopy37 = {
+    version = "2.0.0";
```

Comment:
If they haven't released 2.0.0 you cannot call it that. Please mark it as unstable and add the date as version. See the [manual](https://nixos.org/nixpkgs/manual/#sec-package-naming)

Reply:
Didn't know about the naming convention, sorry.